subsys/net/lib/dhcpv6/dhcpv6.c

```diff
@@ -1327,6 +1380,49 @@ static int dhcpv6_find_status_code(struct net_pkt *pkt, uint16_t *status)
 	return ret;
 }
 
+static int dhcpv6_handle_dns_server_option(struct net_pkt *pkt)
+{
+	const struct sockaddr *dns_servers[MAX_DNS_SERVERS + 1] = { 0 };
+	struct sockaddr_in6 dnses[MAX_DNS_SERVERS] = { 0 };
```

Comment:
The `dnses` variable is a bit funny actually (at least my tongue twists when trying to say it). But perhaps calling it `dns_saddr` or similar would be more descriptive.

Reply:
Updated